openslides_backend/action/actions/user/user_scope_permission_check_mixin.py

```diff
@@ -18,9 +18,9 @@
 
 
 class UserScope(Enum):
```

Comment:
If you change this to `UserScope(int, Enum)` you should be able to use it below with the `.value` suffix

Reply:
You mean, I could use it without `.value` suffix?! Done

Reply:
yes, sorry :D